py3status/modules/google_calendar.py

```diff
@@ -30,24 +30,25 @@
         (default False)
     format: The format for module output.
         (default '{events}|\?color=event \u2687')
-    format_date: The format for date related format placeholders. May be any Python
-        strftime directives for dates.
+    format_date: The format for date related format placeholders.
+        May be any Python strftime directives for dates.
         (default '%a %d-%m')
-    format_event: The format for each event. The information to display for each event
-        can be toggled with 'button_toggle' based on the value of 'is_toggled'.
-        (default  '[\?color=event {summary}][\?if=is_toggled  ({start_time}
-         - {end_time}, {start_date})|[\?if=location  ({location})] [{format_timer}]]'
+    format_event: The format for each event. The information can be toggled
+        with 'button_toggle' based on the value of 'is_toggled'.
+        (default '[\?color=event {summary}][\?if=is_toggled  ({start_time}
+         - {end_time}, {start_date})|[\?if=location  ({location})]
+        [{format_timer}]]'
     format_notification: The format for event warning notifications.
         (default '{summary} {start_time} - {end_time}')
     format_separator: The string used to separate individual events.
         (default ' \| ')
-    format_time: The format for time related format placeholders (except {format_timer}).
+    format_time: The format for time-related placeholders except {format_timer}.
```

Comment:
except \`{format_timer}\`.  

will look nicer in the docs

Reply:
Okay. Coming up soon.